src/compile/render-dom/wrappers/Element/StyleAttribute.ts

```diff
@@ -17,7 +17,6 @@ export default class StyleAttributeWrapper extends AttributeWrapper {
 
 	render(block: Block) {
 		const style_props = optimize_style(this.node.chunks);
-		if (!style_props) return super.render(block);
```

Comment:
`optimize_style` can return `null` (line 73 & 76) and as such, this check is needed.

Reply:
Correct, thanks